SU2_CFD/src/output/COutput.cpp

```diff
@@ -347,15 +356,57 @@ void COutput::WriteToFile(CConfig *config, CGeometry *geometry, unsigned short f
 
     case RESTART_BINARY:
 
-      if (fileName.empty())
+      if (fileName.empty()){
         fileName = config->GetFilename(restartFilename, "", curTimeIter);
 
+        if (!config->GetWrt_Restart_Overwrite()){
+          if (config->GetMultizone_Problem()){
+            /* --- do nothing --- */  
+          }
```

Comment:
Can you append numbers also to multizone problems?

Reply:
yes, it looks like the only difference is that there is already a zone number added.